packages/ui/src/components/editor/panels/Assets/container/index.tsx

```diff
@@ -370,8 +368,6 @@ const AssetPanel = () => {
         key: {
           $like: `%${searchText.value}%`
         },
-        type: 'asset',
```

Comment:
@DanielBelmes @dinomut1 why are we removing the 'asset' type here, we decided that we want a dedicated 'asset' type (that currently auto-populates if the tags field has more than one tag, meaning it was intentially given a tag by the content team beyond the autogenerated file type tag). This allows us to filter out thumbnails, scenes, avatars, recordings etc. Otherwise all of these will populate into the asset panel. We need something that explicitly defines a file is an asset.

Reply:
just to keep a record here. At least for the time being I think it's best to remove this part of the query until we get all the assets tagged as asset.

Reply:
@HexaField "Assets" don't exist right now. We want both files and assets in the assets panel. We only want to hide files when an asset exists that represents said file as a LODded version